core/src/commonMain/kotlin/com/ustadmobile/core/controller/EpubContentPresenter.kt

```diff
@@ -142,6 +145,7 @@ class EpubContentPresenter(context: Any,
         super.onCreate(savedState)
         val containerUid = arguments[UstadView.ARG_CONTAINER_UID]?.toLong() ?: 100
         contentEntryUid = arguments[UstadView.ARG_CONTENT_ENTRY_UID]?.toLong() ?: 0
+        contextRegistration = UMUUID.randomUUID().toString()
```

Comment:
I have removed UMUUID in primary. It was leftover from J2ME days. It's better to use an expect/actual instead. We now have KmpUuid in lib-door-runtime (and the top level function randomUuid() - which is interestingly the example that Kotlin gives for when to use expect/actual - https://kotlinlang.org/docs/mobile/connect-to-platform-specific-apis.html#example-send-and-receive-messages-from-a-websocket ).